src/gui/qgsmaskingwidget.cpp

```diff
@@ -332,3 +332,8 @@ void QgsMaskingWidget::apply()
     layer->triggerRepaint();
   }
 }
+
+bool QgsMaskingWidget::hasBeenPopulated()
+{
+  return mMustPopulate == false;
```

Comment:
```suggestion
  return !mMustPopulate;
```
is a bit more intuitive?

Reply:
Done !